com.wamas.ide.launching/src/com/wamas/ide/launching/generator/StandaloneLaunchConfigGenerator.xtend

```diff
@@ -45,16 +45,6 @@ class StandaloneLaunchConfigGenerator {
 	@Inject
 	extension IQualifiedNameProvider qnp
 
-	// also see org.eclipse.pde.internal.ui.editor.product.PluginConfigurationSection.getBundlesWithStartLevels()
-	// for a list of known default start-levels which should be present.
-	val knownStartLevels = newHashMap(
```

Comment:
why is that logic remove?

Reply:
Because the entries as returned by DependencyResolver.resolveAndExpand() contain proper startlevels per this commit. But then, the workspace may contain such a special bundle, be it for debugging purposes only.
Moving the logic back here, leaving DependencyResolver alone.

Reply:
It turns out that PDE's internal list of well-known bundles with startlevels is eventually incomplete, or at least eventually outdated.
Actually, it's a recommendation only, see also https://github.com/eclipse-pde/eclipse.pde/discussions/754.

Reply:
Preserving that list locally, just updating based on what is found in Eclipse 2023-06.